go.mod

```diff
@@ -0,0 +1,10 @@
+module github/go-wireless
```

Comment:
Should this be `github.com/Plantiga/go-wireless` or maybe `github.com/theojulienne/go-wireless`?

Reply:
I've been working on some other changes to this module that should make the naming more clear.

Right now, our CloudAdapter repo uses a `replace` assignment to redirect all mentions of `github.com/theojulienne/go-wireless` to our plantiga fork--I'm going to refactor this module to use references to our fork to reduce any confusion. 

We will then need to refactor all the references in our CloudAdapter repo once those changes are merged into this repo.